nmdc_server/auth.py

```diff
@@ -250,7 +251,35 @@ async def orcid_authorize(request: Request, db: Session = Depends(get_db)):
     # TODO: use token_response.access_token to make requests to the ORCID API for more user info?
     # Or possibly store the access token in the database for later use?
     user = User(orcid=token_response["orcid"], name=token_response["name"])
+
+    orcid_email_response = await oauth2_client.orcid.get(
+        "https://pub.orcid.org/v3.0/0009-0004-9185-506X/email", token=token_response
+    )
+
+    # XML data from ORCID API
+    xml_content = orcid_email_response._content
+
+    root = ElementTree.fromstring(xml_content)
+
+    namespace = {
+        "email": "http://www.orcid.org/ns/email",
+    }
+
+    email_elements = root.findall(".//email:email", namespace)
+
+    user_email = None
+    # Get the first valid email
+    # TODO - Consider storing all valid emails in the future
+    for email in email_elements:
+        if email.text and "@" in email.text:
+            user_email = email.text
+            break
+
+    user_email = user_email
```

Comment:
Can this line be removed?

Reply:
removed in 3c6ce981d57ae696db0df296997f1ab1e62521ed